apps/framework-cli/src/cli/commands.rs

```diff
@@ -277,4 +277,29 @@ pub enum WorkflowCommands {
         #[arg(long)]
         from: String,
     },
+    /// List running workflows
+    List {
+        /// Filter workflows by status (running, completed, failed)
+        #[arg(short, long)]
+        status: Option<String>,
+
+        /// Limit the number of workflows shown
+        #[arg(short, long, default_value = "10")]
+        limit: u32,
+    },
+    /// Terminate a workflow
+    Terminate {
+        /// Name of the workflow to terminate
+        name: String,
+    },
+    /// Pause a workflow
+    Pause {
+        /// Name of the workflow to pause
+        name: String,
+    },
+    /// Unpause a workflow
+    Unpause {
```

Comment:
When would I use `resume` vs `unpause`?

Reply:
they might be the same, but I used unpause to mirror pause.

Reply:
This functionality is behind a feature flag so it won't be seen by most. Also pause and unpause are just placeholders and lower priority relative to other tasks.  I can remove them for now if anyone feels strongly about them being there.